src/components/CodeSuccessModal/CodeSuccessModal.module.scss

```
@@ -0,0 +1,28 @@
+@import '../../styles/mixins';
+@import '../../styles/var.scss';
+
+.modal_content {
+  @include modalContent;
+  text-align: center;
```

```suggestion
  & {
    text-align: center;
  }
```
i'm seeing Sass working here , suggesting me to `wrap the declaration in "& {}"`